diff.py

```diff
@@ -599,12 +623,22 @@ def ansi_ljust(self, s: str) -> str:
 
 @dataclass
 class HtmlFormatter(Formatter):
+    rotation_formats: int = 9
```

Comment:
I guess it's fine for an embedder for set `HtmlFormatter.rotation_formats = something else`